CMakeLists.txt

```diff
@@ -83,8 +92,12 @@ if(CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
     set(CMAKE_INSTALL_PREFIX "/usr")
 endif()
 
-add_subdirectory(src)
+add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/src)
```

Comment:
Since narurally the `src` exists in `project/src`, I would suggest `PROJECT_SOURCE_DIR` instead of `CURRENT_SOURCE_DIR`

Reply:
Sure, does the same thing here.
I just have a habit of always writing 
`${CMAKE_CURRENT_SOURCE_DIR}/subdir_name`